.github/workflows/ci.yaml

```diff
@@ -0,0 +1,34 @@
+name: CI
+on: [push, pull_request, workflow_dispatch]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repo
+      uses: actions/checkout@v2
+
+    - name: Setup Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.15.3
+     
+    - name: CI - Verifications and Tests
```

Comment:
Consider just dropping the name and following 
```
    - run: make toolchain
    - run: echo "/usr/local/kubebuilder/bin" >> $GITHUB_PATH
    - run: make ci
```

Reply:
fwiw I find the multiple runs harder to read than the pipe syntax